ansible/roles/products/teleop/server/desktop-icons/tasks/real-robots.yml

```diff
@@ -198,7 +198,7 @@
     desktop_icon_png: "{{ teleop_glove_right }}"
     launch_script: "shadow_GUI_right.sh"
     project_name_input: "sr_teleop_vive_polhemus"
-    launch_file_input: "sr_teleop_vive_polhemus.launch external_control_loop:=true vive:=true robot_model:={{ ur_robot_type }} require_pedal:=true biotacs:={{ biotacs | lower }} user_calibration_folder_path:='/home/user/shadow_glove_user_calibration_files' user_calibration_file_name:=current_calibration.yaml"
+    launch_file_input: "sr_teleop_vive_polhemus.launch external_control_loop:=true vive:=true robot_model:={{ ur_robot_type }} require_pedal:=true biotacs:={{ biotacs | lower }} user_calibration_folder_path:='/home/user/shadow_glove_calibration/user_calibration' user_calibration_file_name:=current_calibration.yaml calibration_file_path:='/home/user/shadow_glove_calibration/glove_boresight_calibration' calibration_file_name:=liberty_calibration.yaml"
```

Comment:
remove calibration_file_name var, rename calibration_file_path to boresight_calibration_file_path

Reply:
Fixed in the last commit